x-pack/plugins/uptime/public/components/monitor/ping_list/ping_list_table.tsx

```diff
@@ -140,7 +140,18 @@ export function PingListTable({ loading, error, pings, pagination, onChange, fai
       name: i18n.translate('xpack.uptime.pingList.durationMsColumnLabel', {
         defaultMessage: 'Duration',
       }),
-      render: (duration: number) => formatDuration(duration),
+      render: (duration: number | null) =>
+        duration ? (
+          formatDuration(duration)
+        ) : (
+          <EuiToolTip
+            content={i18n.translate('xpack.uptime.pingList.durationColumnToolTip', {
+              defaultMessage: 'Duration is unavailable for journeys that do not finish executing',
```

Comment:
Will this logic ever apply to lightweight checks? The content I have now is very specific to journeys

Reply:
Would this be better to keep error not specific to journeys? Or even keep it as '--'